Source/Core/Core/Boot/Boot.cpp

```diff
@@ -271,199 +312,187 @@ bool CBoot::Load_BS2(const std::string& boot_rom_filename)
   return true;
 }
 
-// Third boot step after BootManager and Core. See Call schedule in BootManager.cpp
-bool CBoot::BootUp()
+static const DiscIO::IVolume* SetDefaultDisc()
 {
-  SConfig& _StartupPara = SConfig::GetInstance();
+  auto& config = SConfig::GetInstance();
```

Comment:
Well, my reasoning is that by calling GetInstance() on SConfig, it's already obvious what `config` is going to be, so there is in my opinion no point in writing SConfig a second time. It's more about avoiding redundancy than reducing keystrokes. I don't really care about it in this case though, so I've changed it back to SConfig (and also added the const -- since this is new code).